agent/src/util/device_plugin_service.rs

```diff
@@ -453,6 +519,7 @@ async fn try_update_instance_device_usage(
     node_name: &str,
     instance_name: &str,
     instance_namespace: &str,
+    device_usage_kind: DeviceUsageKind,
```

Comment:
if a param of type DeviceUsage is passed in, would we need both device_usage_d and device_usage_kind?  or, can device_usage_kind be parsed from device_usage_id? do we need 2 args for device_usage info?

Reply:
nevermind ... i see.

device_usage_id is used to get current slot usage ... device_usage_kind is the request/desired type.

Reply:
maybe: change device_usage_kind to desired_device_usage_kind

Reply:
rename device_usage_kind to desired_device_usage_kind